contents/blog/customer-churn-analysis-guide.md

```diff
@@ -0,0 +1,296 @@
+
+### Problems with churn rate and alternative methods
+
+Other articles you read about churn rate may introduce alternative ways to measure it lifted directly from [this 2011 article](https://shopify.engineering/defining-churn-rate-no-really-this-actually-requires-an-entire-blog-post) by a data scientist at Shopify.
+
+It's a dense affair that highlights some interesting quirks about churn rate, which I'll quickly summarize:
+
+- Extreme growth can lead to distorted churn figures – such as reporting lower churn when user behavior hasn't actually changed. High-growth startups should be wary as a result.
+
+- Because churn rate measures churn among **existing customers**, it can hide churn problems among new, incoming customers.
+
+The Shopify article explores several complex alternative churn formulas to address these problems. It's a laudable objective, but in my view **totally unnecessary**.
+
+Is churn rate a perfect metric? No. Do perfect metrics exist? Also, no. Our objective here isn't mathematical perfection, it's a comparable figure that's easy to understand. The more complicated a metric becomes, the less likely people will use it or understand it. 
+
+Bottom line: You should **never** rely on just one number to measure success. See also why the whole "one number that matters" is a trap.
+
+> 💡**PostHog Tip:** A funnel analysis tracking signup through activation, and conversion-to-paid is a better way to investigate churn among new users. Read our guide to [building funnels in PostHog](/tutorials/funnels) for more on this.
+
+## Understanding retention rate
+
+So that's churn rate, but what about retention rate and why, as I claimed earlier, is it a more useful metric? (most of the time) 
+
+To calculate your overall customer retention over a period you need to know:
+
+- How many customers you have at the start of a period (A)
+- How many customers you have at the end of a period (B)
+- How many new customers were added during that period. (C)
+
+You can then calculate your retention as a percentage (R) using the following formula:
+
+> ((B - C) / A) * 100 = R
+
+For example, if you have 100 customers at the start of a year, gain 35 new customers during the year and end the year with a total of 80 customers then your retention rate would be:
+
+> ((80-35)/100)*100 = 45% retention
+
+Meaning that each year a little more than half of your customers are churning and your usage as a whole is in decline. This would be a worrying sign!
+
+As you'll know if you didn't skip the intro, retention is just the inverse of churn rate. So, the being the case, why is retention rate **more useful** than churn rate?
+
+Because you can analyze retention using a cohort retention table like this:
+
+![new user retention](../images/blog/retention-vs-churn/retention-table-example-all.png)
+
+I'll explain how to read retention tables in a moment, but here's the rub... retention tables allow you to answer questions like:
```

Comment:
I would expect an explanation on how to read this table immediately below it - maybe put these bullets above the table?